setup.py

```diff
@@ -66,13 +66,9 @@
         'Operating System :: Microsoft :: Windows',
         'Operating System :: POSIX',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
```

Comment:
If 3.5 is not getting tested why is it showing up here? I would vote to keep 3.5 around. 3.0-3.4 is fine to get removed though